coderd/database/queries/quotas.sql

```diff
@@ -18,23 +18,34 @@ INNER JOIN groups ON
 WITH latest_builds AS (
 SELECT
 	DISTINCT ON
-	(workspace_id) id,
-	workspace_id,
-	daily_cost
+	(wb.workspace_id) wb.id,
+	wb.workspace_id,
+	wb.daily_cost
 FROM
 	workspace_builds wb
+-- This INNER JOIN prevents a seq scan of the workspace_builds table.
+-- Limit the rows to the absolute minimum required, which is all workspaces
+-- in a given organization for a given user.
+INNER JOIN
+	workspaces on wb.workspace_id = workspaces.id
+WHERE
+	workspaces.owner_id = @owner_id AND
+	workspaces.organization_id = @organization_id
 ORDER BY
-	workspace_id,
-	created_at DESC
+	wb.workspace_id,
+	wb.created_at DESC
```

Comment:
It might be faster to use `wb.build_number`, since the query planner has access to the `workspace_builds_workspace_id_build_number_key` index.  That could be why it's having to do the 

```
              ->  Sort  (cost=22.66..22.66 rows=2 width=44)
"                    Sort Key: wb.workspace_id, wb.created_at DESC"
```

Reply:
I wondered about that too. I had a specific goal and did not want to start mutating the query too much to solicit feedback on different things.

Can I throw this up in another PR?

Reply:
yeah, that works